docs/design/autoware-interfaces/ad-api/list/api/planning/index.md

```diff
@@ -0,0 +1,96 @@
+# Planning API
+
+- {{ link_ad_api('/api/planning/allow_start') }}
+- {{ link_ad_api('/api/planning/state') }}
+- {{ link_ad_api('/api/planning/factors') }}
+
+## Description
+
+This API manages the behavior that the vehicle plans.
+Applications can notify the vehicle behavior to the people around and visualize it for operator and passengers.
+
+## States
+
+The planning state manages the stop and start of the vehicle.
+Once the vehicle has stopped, the state will be STOPPED.
+After this, when the vehicle tries to start (is still stopped), the state will be STARTING.
+In this state, calling the start API changes the state to MOVING and the vehicle starts.
+This mechanism can add processing such as announcements before the vehicle starts.
+Depending on the configuration, the state may transition directly from STOPPED to MOVING.
+
+![planning-state](./docs/state.drawio.svg)
+
+| State    | Description                                     |
+| -------- | ----------------------------------------------- |
+| STOPPED  | The vehicle is stopped.                         |
+| STARTING | The vehicle is stopped, but is trying to start. |
+| MOVING   | The vehicle is moving.                          |
+
+## Factors
+
+The planning factors is an array of information on the behavior that the vehicle plans.
+Each factor has type shown below, pose, distance from the vehicle head to that pose, status, and detailed data depending on its type.
+See the description of each behavior below for pose and status. And see the description of factor types for detailed data.
+
+![planning-factors](./docs/factors.drawio.svg)
+
+| Factor Type                 | Behavior         | Description                                            |
+| --------------------------- | ---------------- | ------------------------------------------------------ |
+| SURROUND_OBSTACLE           | immediate stop   | There are obstacles nearby.                            |
+| FRONT_OBSTACLE              | planned stop     | There are obstacles ahead.                             |
+| INTERSECTION                | planned stop     | There are obstacles in other lanes in the path.        |
+| CROSSWALK                   | planned stop     | There are obstacles on the crosswalk.                  |
+| REAR_CHECK                  | planned stop     | There are obstacles behind like shoulder checks.       |
```

Comment:
This description suggests that a shoulder check is an obstacle, which is not the case. I'm not sure of a good alternative, but here's one for now.

```suggestion
| REAR_CHECK                  | planned stop     | There are obstacles behind that would be in a human driver's blind spot.       |
```